doc/nrf/known_issues.rst

```diff
@@ -113,6 +113,15 @@ IRIS-2676: Missing support for FOTA on nRF Cloud
 Other issues
 ============
 
+.. rst-class:: v1-7-0 v1-6-0 v1-5-0
+
+NCSDK-11684: Failure loading KMU registers on nRF9160 devices
+  Certain builds will experience problems loading HUK to KMU due to a bug in nrf_cc3xx_platform library prior to
```

Comment:
'One sentence - one line' rule, please do not break sentences over multiple lines, and always start a new sentence from a new line.

The description looks good.